controllers/conduit/proxy.go

```diff
@@ -47,28 +42,28 @@ func (i *Proxy) getEnvVars(allEnv []corev1.EnvVar) []corev1.EnvVar {
 	// else return default envVars
 	env := []corev1.EnvVar{
 		{
-			Name:  proxyEnvConfig,
+			Name:  "NSM_CONFIG_MAP_NAME",
```

Comment:
Because they are only used once. Making variables is better for using multiple times, but it is just troublesome right now to look for which parameter corresponds to which string